src/ipz_parser.cpp

```diff
@@ -631,4 +631,31 @@ types::DbusVariantType IpzVpdParser::readKeywordFromHardware(
     return types::DbusVariantType{
         getKeywordValueFromRecord(l_record, l_keyword, l_recordOffset)};
 }
+
+void IpzVpdParser::updateRecordECC(const auto& i_recordDataOffset,
```

Comment:
I could not understand the use case of this API from the given commit message.

Reply:
updated the use case in commit message